### DISCUSSION TITLE
is there a way to sharpen this text?

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#3989] [Digital Image Processing]
Version: ImageMagick 6.9.10-71 Q16 x86_64 2021-07-06 https://imagemagick.org
Copyright: © 1999-2019 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC OpenMP(3.1)
Delegates (built-in): bzlib fftw fontconfig freetype jng jp2 jpeg lzma openexr pangocairo png tiff x xml zlib

Centos 7.6
![FINAL_REDONE](https://user-images.githubusercontent.com/12046302/127197095-637d6a71-c34d-491e-8a17-3546bf6a2cfb.jpg)

**Comment by fmw42** [comment]
`convert image.jpg -unsharp 0x1 result.png`

![s](https://user-images.githubusercontent.com/13458218/127197979-1c7047c4-7685-4807-807a-73b92cf639fa.png)

See https://imagemagick.org/script/command-line-options.php#unsharp

Adjust arguments as desired

**Reply by srini8080** [reply]
anything that can be used to make the text finer?

**Comment by snibgo** [comment]
> anything that can be used to make the text finer?

Yes. Re-scan at a higher resolution.

**Reply by srini8080** [reply]
Will try that.  Thanks
